README.md

```diff
@@ -11,9 +11,8 @@
 # CyberArk Secrets Provider for Kubernetes
 
 The CyberArk Secrets Provider for Kubernetes enables you to use secrets stored and managed in the CyberArk Vault 
-using DAP or Conjur and consume them in your Openshift application containers. To do so, the CyberArk Secrets 
-Provider for Kubernetes image runs as an init container and provides the Conjur secrets, required by the pod, 
-from DAP or Conjur.
+using DAP or Conjur and consume them in your Openshift application containers. To do so, the Secrets Provider for 
```

Comment:
why did you remove the word `CyberArk`?

Reply:
please get a TW approval on this change (but why is it part of this PR?)

Reply:
You previously commented on the discrepancy between what was in the README and what I wrote in the description and from what I understood you said if I don't like it then change it and so I did. I can make it into a separate PR if that would be better and get TW approval. WDYT?